.circleci/config.yml

```diff
@@ -20,7 +20,7 @@ jobs:
             echo "//registry.npmjs.org/:_authToken=${NPM_AUTH_TOKEN}" > .npmrc
       - restore_cache:
           keys:
-          - v1-dependencies-{{ checksum "yarn.lock" }}
+          - myapp-{{ .Environment.CACHE_VERSION }}-{{ checksum "yarn.lock" }}
```

Comment:
this gives us a way to manually bust the circle CI cache

https://support.circleci.com/hc/en-us/articles/115015426888-Clear-Project-Dependency-Cache